cmd/karavictl/cmd/api/api.go

```diff
@@ -312,8 +312,19 @@ func (c *client) DoAndGetResponseBody(
 // ParseJSONError parses the error from the proxy server
 func (c *client) ParseJSONError(r *http.Response) error {
 	jsonError := web.JSONError{}
-	if err := json.NewDecoder(r.Body).Decode(&jsonError); err != nil {
-		return err
+
+	b, err := io.ReadAll(r.Body)
```

Comment:
I have this changes in my PR

Reply:
Reverted to original and will let your PR handle it.